<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSF2PlayerForm</class>
 <widget class="QDialog" name="CSF2PlayerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>156</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(230, 230, 230, 255), stop:0.8 rgba(200, 200, 200, 255));}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="4">
    <widget class="QSynthPanel" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Volume</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" alignment="Qt::AlignHCenter">
       <widget class="QToolButton" name="PatchButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>18</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Patch Change</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="4" alignment="Qt::AlignHCenter">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Source</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QComboBox" name="BankList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="LoadButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QComboBox" name="PresetList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QSynthKnob" name="VolDial">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSynthPanel</class>
   <extends>QFrame</extends>
   <header>qsynthpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSynthKnob</class>
   <extends>QDial</extends>
   <header>qsynthknob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
